plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -98,17 +106,42 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 	return authorizer.DecisionNoOpinion, "", nil
 }
 
+// authorizeStatusUpdate authorizes get/update/patch requests to status subresources of the specified type if they are related to the specified node
+func (r *NodeAuthorizer) authorizeStatusUpdate(nodeName string, startingType vertexType, attrs authorizer.Attributes) (authorizer.Decision, string, error) {
+	switch attrs.GetVerb() {
+	case "get", "update", "patch":
```

Comment:
why "get"?

Reply:
Question - why should GETing status subresource be denied from node while patching or updating is fine? I did not understand security or other implication of allowing nodes to GET status subresource.

Reply:
Nothing in the go client will call "get" on the status subresource of a pod. By principal of least privilege, we should only allow an action if something is actually going to and needs to use it.

Reply:
Hmm, I am not sure I understand. Sticking to PVC in this case(not pods). Node can anyways GET PVC object which includes status subresource. If nodes weren't able to GET PVC object with status field then it will not be possible to PATCH the subresource.

Reply:
nodes call get on the resource which returns the full object. e.g:

GET /api/v1/namespaces/default/pods/pod-1

nodes will never call get on the subresource. e.g:

GET /api/v1/namespaces/default/pods/pod-1/status

Reply:
fixed